core/src/com/unciv/models/Religion.kt

```diff
@@ -7,58 +7,65 @@ import com.unciv.models.stats.INamed
 
 /** Data object for Religions */
 class Religion() : INamed {
-    
+
+    override lateinit var name: String
+    var iconName: String = "Pantheon"
+    lateinit var foundingCivName: String
+    var holyCityId: String? = null
+
+
     var pantheonBeliefs: HashSet<String> = hashSetOf()
     var founderBeliefs: HashSet<String> = hashSetOf()
     var followerBeliefs: HashSet<String> = hashSetOf()
```

Comment:
I don't think these should be different. There should only be one hashset, "beliefs". Each belief can only be on one type anyway, as specified in the json.

Reply:
Merging pantheons and follower beliefs is fine in my opinion, but just for not having to look it up, keeping founder beliefs separate makes a lot of sense. Follower beliefs & pantheon beliefs apply to each city following the religion, while founder beliefs specifically only apply to the founder, and so they would be checked in vastly different circumstances